File: JAVA/2419. Longest Subarray With Maximum Bitwise AND.java
class Solution {
    public int longestSubarray(int[] n) {
        int m=-1,a=0,k=0,i;
        for(i=0;i<n.length;i++){
            if(m<n[i])
            m=n[i];
        }
        for(i=0;i<n.length;i++){
            if(n[i]==m)
            a++;
            else
            a=0;
            if(a>k)
            k=a;
            
        }
        return k;
    }
}

File: C++/2419. Longest Subarray With Maximum Bitwise AND.cpp
class Solution {
public:
    int longestSubarray(vector<int>& n) {
        int m,a=0,k=0;
        m=*max_element(n.begin(),n.end());
        for(int i:n){
            if(i==m)
            a++;
            else
            a=0;
            k=max(a,k);
        }
        return k;
    }
};